django_redis/client/sharded.py

```diff
@@ -258,7 +260,8 @@ def incr_version(self, key, delta=1, version=None, client=None):
             raise ConnectionInterrupted(connection=client) from e
 
         if value is None:
-            raise ValueError("Key '%s' not found" % key)
+            err_msg = f"Key '{key}' not found"
```

Comment:
oops, sorry for this, I've already taken care of it in another PR

Reply:
No worries, I'm glad that it was fixed.